### PR TITLE
Update docker-tutorial.md

### PR DESCRIPTION
**Pull request by TimothyMalahy**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@TimothyMalahy : Thanks for your contribution! The author(s) have been notified to review your proposed change. @nebuk89

File: docker/tutorials/docker-tutorial.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ If you've already run the command to get started with the tutorial, congratulati
 docker run -d -p 80:80 docker/getting-started
 ```
 
+> [!TIP] Make sure you are using proxy settings in Docker if necessary
+> 
+> This can be done by navigating to docker settings
+> 
+> [![](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggVERcbiAgICBBW0RvY2tlciBEZXNrdG9wXSAtLT5CW1NldHRpbmdzXVxuICAgIEIgLS0-IENbUmVzb3VyY2VzXVxuICAgIEMgLS0-RFtQUk9YSUVTXVxuICAgIEQgLS0-RVtUb2dnbGUgJ01hbnVhbCBwcm94eSBjb25maWd1cmF0aW9uJyB0byBvbl0iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcbiAgICBBW0RvY2tlciBEZXNrdG9wXSAtLT5CW1NldHRpbmdzXVxuICAgIEIgLS0-IENbUmVzb3VyY2VzXVxuICAgIEMgLS0-RFtQUk9YSUVTXVxuICAgIEQgLS0-RVtUb2dnbGUgJ01hbnVhbCBwcm94eSBjb25maWd1cmF0aW9uJyB0byBvbl0iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
```
Comment (ghogen) [inline docker/tutorials/docker-tutorial.md:31]: We wouldn't want to include an image link from an exernal site, but we could download an image and check it into our repo.

**Comment by ghogen** [comment]
@TimothyMalahy Thanks for your contribution. I'll take this up as an internal pull request and close this one. That way, I can add the screenshot from Docker Desktop. We should also be more specific about what conditions require the use of a proxy. Your Tip says "if necessary" but doesn't say when that would be helpful or necessary.

File: docker/tutorials/docker-tutorial.md (hunk 1 above)
Comment (ghogen) [reply docker/tutorials/docker-tutorial.md:31]: When is the proxy setting necessary?

Comment (TimothyMalahy) [reply docker/tutorials/docker-tutorial.md:31]: Good point on this. It was necessary for me internally at my company. I'm not educated enough on docker nor proxy to identify that though. Just thanking you for the feedback as I'm obviously new to this stuff and it helps me immensely understand the types of questions that are to be expected in the future and I can use.

**Pull request closed without merging**